app/views/user/goals/index.html.erb

```diff
@@ -30,22 +30,33 @@
       </div>
 
       <div class="goal-second-row">
-        <div class="add-tally"></div>
-        <div class="subtract-tally"></div>
-        <div class="progress-bar">
-          <div class="completed-bar"></div>
-        </div>
+        <!-- <div class="add-tally"></div> -->
+        <!-- <div class="subtract-tally"></div> -->
+        <!-- <div class="progress-bar"> -->
+          <!-- <div class="completed-bar"></div> -->
+        <!-- </div> -->
         <% if  (goal.total_goal_count == goal.progress_count)%>
```

Comment:
same here, this should probably not live in the view..you could have a method on "Goal" that does this for you..

```
class Goal
  def achieved?
    total_goal_count == progress_count
  end
end
```

then, in here you would just have `<% if goal.achieved? %>`